### PR TITLE
Add Settings tab to the user profile page

### PR DESCRIPTION
**Pull request by pazoozoo42**
- move Exit and theme toggle to the user profile settings tab
 - show number of posts, comments and available invites
 - fix a couple of lint warnings

### CODE REVIEW

**Comment by Aivean** [comment]
One useful usecase that we've lost here is changing the theme to see how post preview looks like with different background. Perhaps we should add a small change "theme" button near the "preview" button of the content editor?

File: frontend/src/Components/UserProfileInvites.tsx [hunk 1]
```diff
@@ -63,6 +64,8 @@ export const UserProfileInvites = observer((props: UserProfileInvitesProps) => {
             const result = await api.inviteAPI.create(reason);
             console.log('CREATE', result);
             forceRefresh();
+            props.onInvitesChange();
+            // eslint-disable-next-line
```
Comment (Aivean) [inline frontend/src/Components/UserProfileInvites.tsx:68]: Why are these necessary?

**Review by Aivean — APPROVED** [review]
LGTM

File: frontend/src/Components/UserProfileInvites.tsx (hunk 1 above)
Comment (pazoozoo42) [reply frontend/src/Components/UserProfileInvites.tsx]: you cannot type exception callback argument (typescript limitation) but lint is complaining about it

**Comment by pazoozoo42** [comment]
> One useful usecase that we've lost here is changing the theme to see how post preview looks like with different background. Perhaps we should add a small change "theme" button near the "preview" button of the content editor?

we can but this will switch theme permanently and might cause some irritation if they forget to switch it back after post is created and they will need to go to settings to change it back

**Comment by Aivean** [comment]
> > One useful usecase that we've lost here is changing the theme to see how post preview looks like with different background. Perhaps we should add a small change "theme" button near the "preview" button of the content editor?
> 
> we can but this will switch theme permanently and might cause some irritation if they forget to switch it back after post is created and they will need to go to settings to change it back

We can remember the original settings and switch it back automatically when reply is posted.

**Comment by pazoozoo42** [comment]
> We can remember the original settings and switch it back automatically when reply is posted.

@Aivean https://github.com/spaceshelter/orbitar/pull/217
